LICENSE

```diff
@@ -1,24 +0,0 @@
-Copyright (c) 2015, stack
```

Comment:
LICENSE file in the root is a pretty well accepted practice in the Haskell package world, I'd rather not change that.